### PR TITLE
docs(readme): add beta warning

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

**Review by lalli-flores — CHANGES_REQUESTED** [review]
I was working my way around the component system repos. Can you follow the same wording as my opened PR?
https://github.com/webex/component-adapter-interfaces/pull/41/commits/73f012d438f580754401abd03a168c8587dca73b

**Comment by adamweeks** [comment]
Apologies for creating a branch on the main repo, I used the github web editor to do this and forgot it wasn't on my fork.

File: README.md [hunk 1]
```diff
@@ -20,6 +20,15 @@
...
 aimed at developers that want to embed Webex into their applications.
 
+## Project Status
+
+The Webex Component System is considered to be in beta stage and it's not a generally available product from Webex at this time.
+This means that the Webex Component System is available for everyone to use but breaking changes may occur as we develop it.
+We try our best to minimize any breaking changes but they may occur.
+While the Webex Component System is not a GA product, we still offer support through all regular channels.
+However, bug priority is given to product already generally available.
+We love for you to use the Webex Component System and be part of the feedback process!
```
Comment (lalli-flores) [inline README.md:30]: I actually took your suggestion lol
```suggestion
We would love for you to use the Webex Component System and be part of the feedback process!
```

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**